pyppeteer/page.py

```diff
@@ -85,113 +66,115 @@ class Page(EventEmitter):
     )
 
     @staticmethod
-    async def create(client: CDPSession, target: 'Target',
-                     ignoreHTTPSErrors: bool, defaultViewport: Optional[Dict],
-                     screenshotTaskQueue: list = None) -> 'Page':
+    async def create(
+            client: CDPSession,
+            target: 'Target',
```

Comment:
AFAICT, a forward reference isn't needed because of the `if TYPE_CHECKING` guard
```suggestion
            target: Target,
```